types/stringmappings.pp

```diff
@@ -0,0 +1,4 @@
+type Nginx::StringMappings = Variant[
```

Comment:
can you add a very brief summary for puppet-strings at the top?

Reply:
Ah 🤦🏻‍♂️ sure